### PR TITLE
Kevintest

### PR DESCRIPTION
**Pull request by kevin6040**
Fixed some parts of the pop up view for prosocial or antisocial button

### CODE REVIEW

File: fitIn/popUpViewController.swift [hunk 1]
```diff
@@ -10,19 +10,21 @@ import UIKit
 
 class popUpViewController: UIViewController {
 
-   private var scenarioController = ScenarioHandler()
```
Comment (schecko) [inline fitIn/popUpViewController.swift:13]: where did this go? is it not used?

File: fitInTests/ScenarioTest.swift [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        super.setUp()
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+    
+    func testExample() {
```
Comment (schecko) [inline fitInTests/ScenarioTest.swift]: remove

File: fitInTests/ScenarioTest.swift [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct results.
+    }
+    
+    func testPerformanceExample() {
```
Comment (schecko) [inline fitInTests/ScenarioTest.swift]: remove

File: fitInTests/ScenarioTest.swift [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        // This is an example of a performance test case.
+        self.measure {
+            // Put the code you want to measure the time of here.
+        }
+    }
+    func testCreateScenario(){
```
Comment (schecko) [inline fitInTests/ScenarioTest.swift]: replace this with the type checking, checking for default values is going to cause headaches when I want to change the default values.

File: fitIn/popUpViewController.swift (hunk 1 above)
Comment (kevin6040) [reply fitIn/popUpViewController.swift:13]: I dont neeed it in the pop up view, just the original scenario handler view. The way I have it set up is the scenario handler passes the answer reason to the pop up view

File: fitInTests/ScenarioTest.swift [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+//  Created by Kevin C on 2017-11-03.
+//  Copyright © 2017 AJ productions. All rights reserved.
+//
+
+import XCTest
+@testable import fitIn
+
+class ScenarioTest: XCTestCase {
+    //create a scenario we will fudge with
+    let testScenario = Scenario()
```
Comment (schecko) [inline fitInTests/ScenarioTest.swift]: Create a new scenario for each test case. In these tests all we check for is types but if there are any other types of tests then they will be order dependent on which tests run first.

**Review by schecko — APPROVED** [review]

**Pull request merged**